src/main/java/com/xiaomi/infra/pegasus/tools/interceptor/TableInterceptor.java

```diff
@@ -0,0 +1,13 @@
+package com.xiaomi.infra.pegasus.tools.interceptor;
+
+import com.xiaomi.infra.pegasus.base.error_code.error_types;
+import com.xiaomi.infra.pegasus.rpc.async.ClientRequestRound;
+import com.xiaomi.infra.pegasus.rpc.async.TableHandler;
+
+public interface TableInterceptor {
+  // The behavior before the ReplicaSession sends the RPC.
```

Comment:
```suggestion
  // The behavior before sending the RPC to a table.
```

This interface is not limited to ReplicaSession. An RPC could reach out to multiple replica nodes, or meta nodes.